app/os_ios.go

```diff
@@ -86,10 +86,17 @@ import (
 	"gioui.org/unit"
 )
 
-type ViewEvent struct{}
+type ViewEvent struct {
+	// View is a CFTypeRef for the NSView for the window.
+	View uintptr
+
+	// Window is a CFTypeRef for the NSView for the window.
+	Window uintptr
```

Comment:
Window is a confusing name for a NSView. Can't client code fetch the parent NSView (or NSWindow) through ViewEvent.View?

Reply:
I think it's possible. At least, using `[view superview]` seems to work. _Considering that `view` is the C.CFTypeRef  of `ViewEvent.View`._